sympy/geometry/util.py

```diff
@@ -623,7 +623,6 @@ def idiff(eq, y, x, n=1):
         derivs[dydx] = yp
         eq = dydx - yp
         dydx = dydx.diff(x)
-  
 def intersection(*entities, **kwargs):
     """The intersection of a collection of GeometryEntity instances.
```

Comment:
@smichr I'll do that. Anything else to be done?

_Sent from my LENOVO Lenovo P1a42 using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_